src/main/java/com/shaft/driver/internal/DriverFactoryHelper.java

```diff
@@ -59,7 +59,7 @@ public class DriverFactoryHelper {
     private static String TARGET_HUB_URL;
     private static Boolean MOBILE_EMULATION;
     private static Boolean MOBILE_EMULATION_CUSTOM_DEVICE;
-    private static Boolean LIGHTHOUSE_EXEUTION;
+    private static Boolean LIGHTHOUSE_EXECUTION;
```

Comment:
@Mohab-Osama @mohabosamaa we need to add the lighthouse execution parameter in the related properties manager class and the configuration manager html.